libraries/transport/transportControl/transportControlI.H

```diff
@@ -108,13 +111,52 @@ void Foam::Pmt::transportControl<Base>::update()
 {
     if (!upToDate_)
     {
-        scalar maxDeltaC;
+        /*scalar maxDeltaC;
         bool maxDeltaCSet = this->controlDict().readCheckIfPresent
             (
                 "maxDeltaC", 
                 maxDeltaC,
                 [](scalar v){ return v>=0; }
             );
+        */
+
+        bool maxDeltaCSet = false;
+        List<scalar> maxDeltaC(composition_.species().size(), std::numeric_limits<scalar>::infinity());
+
+        const dictionary& maxDeltaCDict = this->controlDict().subOrEmptyDict("maxDeltaCValues");
```

Comment:
Hold on, I think this can be something like `const dictionary* maxDeltaCDict = this->controlDict().findDict("maxDeltaC")`. I like the name `maxDeltaC` (same as for the "universal" value) for the dict more than `maxDeltaCValues`, what do you think?

Reply:
Unless you’ve already tried it and it didn’t work…